Makefile

```diff
@@ -0,0 +1,4 @@
+rpm:
+	python3 setup.py sdist
+	cp dist/* ~/rpmbuild/SOURCES/
+	rpmbuild -ba osbuild.spec
```

Comment:
We usually try to keep away from rpm's `~/rpmbuild` to make sure that we don't accidentally use something from another build. Something like this: https://github.com/weldr/cockpit-composer/blob/master/Makefile#L54